jupyter_book/book_template/_includes/analytics.html

```diff
@@ -0,0 +1,11 @@
+{% if site.google_analytics.mytrackingcode %}
```

Comment:
could this be `googleanalytics.html` just to be more explicit?

Reply:
sure!